torch/autograd/profiler.py

```diff
@@ -1226,20 +1229,21 @@ def parse_kineto_results(result):
     # output top-level memory events
     for mem_record in mem_records:
         if not mem_record[1]:
-            rel_start_us = mem_record[0].start_us() - start_record.start_us()
+            rel_start_us = mem_record[0].start_us() - trace_start_us
+            max_evt_id += 1
             fe = FunctionEvent(
-                id=mem_record[0].handle(),
+                id=max_evt_id,
                 name="[memory]",
                 trace_name=None,  # not outputting in the trace
-                thread=mem_record[0].thread_id(),
+                thread=mem_record[0].start_thread_id(),
                 start_us=rel_start_us,
                 end_us=rel_start_us,  # no duration
-                fwd_thread=mem_record[0].fwd_thread_id(),
+                fwd_thread=mem_record[0].start_thread_id(),
                 input_shapes=[],
                 stack=[],
-                scope=mem_record[0].scope(),
-                cpu_memory_usage=mem_record[0].cpu_memory_usage(),
-                cuda_memory_usage=mem_record[0].cuda_memory_usage(),
+                scope=0,  # function
```

Comment:
Can you add a comment to point to `aten/src/ATen/record_function.h` where RecordScope is and 0 = Function?